packages/next/build/webpack-config.ts

```diff
@@ -60,11 +60,24 @@ function getOptimizedAliases(isServer: boolean): { [pkg: string]: string } {
   }
 
   const stubWindowFetch = path.join(__dirname, 'polyfills', 'fetch.js')
+  const stubWindowObjectAssign = path.join(
+    __dirname,
+    'polyfills',
+    'object-assign.js'
+  )
+
   return {
     __next_polyfill__fetch: require.resolve('whatwg-fetch'),
     unfetch$: stubWindowFetch,
     'isomorphic-unfetch$': stubWindowFetch,
     'whatwg-fetch$': stubWindowFetch,
+
+    // Alias Object.assign
+    '__next_polyfill_object-assign': require.resolve('object-assign'),
+    'object-assign$': stubWindowObjectAssign,
+    'object.assign$': stubWindowObjectAssign,
+    'object.assign/polyfill$': stubWindowObjectAssign,
```

Comment:
Per above, this needs to be a fn export:
```js
module.exports = function getPolyfill() {
  return Object.assign;
}
```

Reply:
We may as well stub out `object.assign/shim$` while we're at it.

Reply:
`object.assign/auto` should also be noop'd. Just an empty file will do.

Reply:
I quite dislike this module